astropy/timeseries/sampled.py

```diff
@@ -150,28 +150,47 @@ def fold(self, period=None, midpoint_epoch=None):
         ----------
         period : `~astropy.units.Quantity`
             The period to use for folding
-        midpoint_epoch : `~astropy.time.Time`
+        epoch : `~astropy.time.Time`
             The time to use as the midpoint epoch, at which the relative
-            time offset will be 0. Defaults to the first time in the time
-            series.
+            time offset / phase will be 0. Defaults to the first time in the
+            time series.
+        normalize_phase : bool
+            If False phase is returned as `~astropy.time.TimeDelta`, otherwise
+            as dimensionless `~astropy.units.Quantity`.
```

Comment:
```suggestion
            as a dimensionless `~astropy.units.Quantity` in the range [0:1].
```

Reply:
In fact it's not in the range of [0:1], +-0.5 from the epoch_phase, so by default -0.5..0.5. 

It seemed nicer this way, but I'm not at all sure about this approach.

Reply:
I thought normalized phases were generally expressed in the range [0:1] so that it'd make more sense to set ``epoch_phase`` to 0.5 and have the final range be [0:1]?

Reply:
OK, i'm indeed changing `epoch_phase` to 0.5, but I still feel it could make sense to have the range as -0.5..0.5, so would not enforce the range of [0:1]?